Software/requirements.txt

```diff
@@ -0,0 +1,9 @@
+setuptools
+pip
+python-dev
+linux-headers
+python3>=3.4,<3.5

```

Comment:
None of the packages above here belong in requirements.txt.

You're writing a [pip requirements file](https://pip.pypa.io/en/stable/user_guide/#requirements-files), and pip is the package manager for python. That means that the only things we can specify are things that pip can install for us.

Finally, please review the best practices [here](https://devcenter.heroku.com/articles/python-pip) and implement them below.